.eslintrc.yml

```diff
@@ -43,9 +42,12 @@ rules:
   no-new: "off"
   no-plusplus: "off" 
   no-multi-str: "off"
-  mocha-no-only/mocha-no-only: ["error"]
+  no-unused-vars: "off"
```

Comment:
Why is this "off"? Does it cause conflicts with `@typescript-eslint/no-unused-vars`?

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md